lightkurve/io/tasoc.py

```diff
@@ -0,0 +1,48 @@
+"""Reader for A PSF-Based Approach to TESS High Quality Data Of Stellar Clusters (PATHOS) light curve files.
+
+Website: https://archive.stsci.edu/hlsp/pathos
+A product description file wasn't obvious on the MAST website
+"""
```

Comment:
☝️ We will want to update the comment section above to explain what TASOC is and where the product is documented.

Reply:
Added 
"""TESS Asteroseismic Science Operations Center - https://tasoc.dk
   TESS Data For Asteroseismology Lightcurves - https://archive.stsci.edu/hlsp/tasoc 
   Data provided with this release have been extracted using the TASOC Photometry pipeline. The TASOC
   pipeline used to generate the data is open source and available on GitHub - https://github.com/tasoc
"""